VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -284,11 +290,11 @@ export abstract class WaitingRoomBaseComponent {
     }
 
     call() {
-        this.logger.info('[WR] - calling pexip');
+        this.logger.info(`${this.loggerPrefix} calling pexip`);
         const pexipNode = this.hearing.getConference().pexip_node_uri;
         const conferenceAlias = this.hearing.getConference().participant_uri;
         const displayName = this.participant.tiled_display_name;
-        this.logger.debug(`[WR] - Calling ${pexipNode} - ${conferenceAlias} as ${displayName}`);
+        this.logger.debug(`${this.loggerPrefix} Calling ${pexipNode} - ${conferenceAlias} as ${displayName}`);
```

Comment:
could you add the conference and participant as the third param like the other logs do please

Reply:
added conference id and participant id